TESTING.md

```diff
@@ -0,0 +1,81 @@
+# Testing
+
+We use automated tests to maintain quality in our codebase. Here's an overview of the kinds of tests we use, and how to run them.
+
+## Kinds of tests
+* Apps directory
+  * Mocha Tests - Used to test client-side functionality of some of our levels, applab, and applab controls
+* Dashboard directory
+  * Ruby tests - All of the server side business logic testing is through here. 
+  * Konacha Tests - Subset of client-side functionality testing is here, we use this to test storage of client state.
+  * UI tests - Used to test overall functionality. Intended to run across browsers. Runs either on Saucelabs or with a local Chromedriver
+    * Eyes tests - Subset of UI tests intended to test the precise layout of controls on certain UI pages. Eyes tests are run through Applitools and work by comparing an expected screenshot to an actual screenshot of a certain page. Eyes tests only run on Chrome for now. If you make a change that affects layout, you will likely break eyes tests. Work with whoever is reviewing your PR to figure out if the layout change should be accepted, and the baseline will be adjusted.
+ * Shared directory
+   * Ruby tests - Unit tests over Ruby code in the shared directory.
+
+## Running tests
+
+### Mocha Tests
+`grunt exec:mocha test` will run all mocha tests. Run this from the `apps` directory
```

Comment:
Do you run our apps tests this way?  I always use `grunt mochaTest` instead of `grunt exec:mocha test`.